test/modules/core/lib/layer-manager.spec.js

```diff
@@ -138,6 +144,22 @@ test('LayerManager#setLayers', t => {
       dataChanged: true,
       propsChanged: false
     },
+    {
```

Comment:
It would probably be nice to at least add a comment before each case explaining what it tests. It won't be easy to figure out when reading the entire file later on.